src/utils.test.js

```diff
@@ -0,0 +1,22 @@
+import { getTimestring, getDatestring, getBackgroundColor } from "./utils";
+
+describe("utils", () => {
+  describe("getTimestring", () => {
+    it("renders correct timestring given a date", () => {
+      const date = new Date("2019-11-12T12:34:11Z");
```

Comment:
I'd add a new line between the setup and expects in a test file. Makes the code more readable. But otherwise good first stab at testing.